ProjFS.Mac/PrjFSLib/PrjFSLib.cpp

```diff
@@ -657,6 +666,14 @@ static void HandleKernelRequest(void* messageMemory, uint32_t messageSize)
                 << errno
                 << "(" << strerror(errno) << ")"
                 << endl;
+            string errorMessage = ss.str();
+
+            s_callbacks.LogError(errorMessage.c_str());
+            cout << errorMessage << endl;
```

Comment:
Now that we're sending this message to the provider, should we still be writing it to the console?

Reply:
I was debating that too.  Since we don't need it i'll remove the cout.